### PR TITLE
MKL: cifar 10 divergance fix and batchnorm unit test fix

### PR DESCRIPTION
**Pull request by gzmkl**
1. Fix Cifar 10 diverging issue
2. Fix BatchNorm unit test failures

### CODE REVIEW

**Review by tatianashp — CHANGES_REQUESTED** [review]
Thank you for fixing divergence issues and test failures! Code readability is very important to us. Could you please edit the comments to be grammatically correct? If you want to abbreviate TensorFlow please use TF not tf.

**Comment by gztf** [comment]
Based on suggestion from Tatiana (tatianashp), changes have been made to fix improper comments such as replacing "tf" with "TensorFlow".  The fix touches <comment> statements only - so no extra test is needed.

**Review by tatianashp — APPROVED** [review]
Thanks for addressing the tf abbreviation issues. I'll fix the rest after the PR goes through.
